app/src/main/java/org/schabi/newpipe/player/BasePlayer.java

```diff
@@ -275,6 +277,7 @@ public void handleIntent(Intent intent) {
         final float playbackPitch = intent.getFloatExtra(PLAYBACK_PITCH, getPlaybackPitch());
         final boolean playbackSkipSilence = intent.getBooleanExtra(PLAYBACK_SKIP_SILENCE,
                 getPlaybackSkipSilence());
+        final boolean isMuted = intent.getBooleanExtra(IS_MUTED, isMuted());
```

Comment:
To be honest I haven't thought about that risk while writting it. And now I think there is no "null risk"... I think so because handleIntent() is only called when intent is received -> which happens only while switching between base/popup/bckgrnd player -> which have to be != null ass Intent comes from other player :) But maybe I am missing something. Please correct me if I am wrong.

Reply:
No, that's reasonable, you are right. But I'd put a null check anyway just to be sure, something like ()```suggestion
        final boolean isMuted = intent.getBooleanExtra(IS_MUTED, simpleExoPlayer == null ? false : isMuted());
```